fastlane/env_vars_example

```diff
@@ -12,6 +12,9 @@ export APP_SCHEME=mattermost
 #export VERSION_NUMBER=
 ## This sets the version number ex: 210 if not set it uses the one in the code base (this one should increment always by 1 and will only update if INCREMENT_BUILD_NUMBER is true
 #export BUILD_NUMBER=
+## This sets the build tasks for the Android. This is a comma-separated list of tasks that can have 2 values - 'assemble' and 'bundle'.
+## If not set, it will use only the 'assemble' task. Add both values to the list if you want both apk and aab files. Example is given below
+#export ANDROID_BUILD_TASK=assemble, bundle
```

Comment:
this is great, but it should also be included in this doc https://github.com/mattermost/mattermost-developer-documentation/blob/master/site/content/contribute/mobile/build-your-own/android.md

Reply:
@cwarnermm can you please review this text before @manojmalik20 submits a PR to the developer docs, thank you!

Reply:
@enahum @manojmalik20 Feedback provided inline. Thanks!